src/frontend/Module.php

```diff
@@ -97,7 +101,20 @@ public function registerComponents()
             ],
         ];
     }
-    
+
+    public function luyaBootstrap(Application $app)
+    {
+        if (!$app->request->isConsoleRequest && $app->has('themeManager')) {
+            // set active theme from database
+            $app->get('themeManager')->on(ThemeManager::EVENT_BEFORE_SETUP, function (SetupEvent $event) {
+                $activeTheme = Theme::findOne(['is_active' => 1]);
```

Comment:
@boehsermoe maybe we should also exclude admin requests? also we might cache this request as it will run on every request, maybe we don't need the active record only to retrieve base_path this will reduce memory usage (asArray() or even better use scalar())

Reply:
Admin requests should exclude anyway because it is in the frontend module class, or I am wrong?

Reply:
As they are bootstraped, they will run on every page request whether its admin or frontend.

Reply:
So you will need to add: !$request->isAdminRequest

Reply:
Solved